pyhms/sprout/sprout_filters.py

```diff
@@ -64,9 +64,6 @@ def __call__(
         candidates: dict[AbstractDeme, DemeCandidates],
         tree,
     ) -> dict[AbstractDeme, DemeCandidates]:
-        assert all(
-            [cand.features.nbc_mean_distance is not None for cand in candidates.values()]
-        ), "NBC_FarEnough filter requires nbc_mean_distance feature in candidates added through NBC_Generator"
```

Comment:
The assert went away after all :D

Reply:
I've seen this change and I thought that I fixed it. Sorry for this.